### ISSUE TITLE
Grey-Green Dailies Hard to Distinguish from Disabled Ones

### ISSUE THREAD

**Issue by ArlieS** [HabitRPG/habitica#2325]
This morning there appears to be an overall colour change in my ToDo/daily/habit list. Blues and greens have more grey in them. The result is that some dailies are pretty close to grey - even though they are neither checked off nor turned off for the day. 

I can tell the difference if I look closely, but I can't see which ones are still active at a glance like I could as recently as yesterday. 

I fear someone has tried to improve aesthetics at the expense of functionality. 

I do not believe my browser has been updated overnight - I'm on linux, not windows, so I have a bit of control over when updates happen :-( The other possibile local-to-me explanation is some bizarre effect of system load - I'm getting overall bad performance this morning, particularly in firefox, including complaints about Habit's script(s) running too long. (And yes, I cleared out my completed ToDos recently.) 

It's also possible the nearly-grey dailies just hit some threshold. They are all recently created, with streaks in the 5-8 day range. Perhaps that state has had a hard-to-see colour for a long time, and I simply haven't had tasks in that range. But my yellow habits also seem to have changed colour - they are quite faded. And probably the blue dailies also, though I'm less sure of that. 

![screenshot from 2014-01-09 08 47 46](https://f.cloud.github.com/assets/4530463/1879664/50806a64-794d-11e3-9857-bc096a5e5a01.png)

**Comment by wc8** [comment]
I do not see any change on my end. Are you still experiencing this?

**Comment by ArlieS** [comment]
Wow - this is bizarre. The greyness seems to have reduced noticeably, and I haven't done anything interesting like restarting the browser - just killed off a few browser windows, trying to get to a point where doing a kill-and-restart is reasonable. But now the colours in the screenshot above don't match the ones in my habit window. 

Attaching new screenshot
![screenshot from 2014-01-09 10 04 24](https://f.cloud.github.com/assets/4530463/1880414/076d0822-7958-11e3-8e58-2e2f443f5de8.png)

**Comment by SyntaxPickax** [comment]
Looking at your screenshots, the colors of everything including the browser toolbar icons appear to have been affected. I think it's safe to say that HabitRPG isn't that cause of this.  Loose monitor cable perhaps?

**Comment by ArlieS** [comment]
Hmm - the screenshot should have picked up what the graphics card intended to display, not what I actually saw, since it's taken from the same comp - not with a camera. But that doesn't rule out it being something internal to my system. You've got good eyes to spot differences in the toolbar - I'm not sure whether I see them even once you mention them. However, I wouldn't put it past my window manager to have decided to grey things slightly for reasons known only to it - Ubuntu's Unity desktop has a mind of its own, and a notable lack of documentation. It _does_ fade things when they aren't responding - but firefox and habitrpg were both responding at the time I took the first screen shot.

Looks like we can close this, and blame Unity. One more reason for me to switch window managers. _sigh_
